addon/components/yeti-table/component.js

```diff
@@ -229,17 +229,30 @@ class YetiTable extends DidChangeAttrsComponent {
    * clicking on the table headers. You can either pass in a comma-separated string or an array
    * of strings. Accepted values are `'asc'`, `'desc'` and `'unsorted'`. The default value is `['asc', 'desc']`.
    */
-  @argument(unionOf('string', arrayOf('string')))
-  sortSequence = ['asc', 'desc'];
+  @argument(optional(unionOf('string', arrayOf('string'))))
+  sortSequence = this.get('config').sortSequence || ['asc', 'desc'];
 
   @className
   @reads('mergedTheme.table')
   themeClass;
 
+  // If the theme is replaced, this will invalidate, but not if any prop under theme is changed
+  @computed('theme')
```

Comment:
Better to add the `config.theme` key, just to be safe and explicit.

Reply:
Since I doubted that config.theme could really change during the course of the application, figured there was no reason to set an observer up on it, but will add.